cmd/npmExecuteLint_test.go

```diff
@@ -0,0 +1,363 @@
+package cmd
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"github.com/SAP/jenkins-library/pkg/log"
+	"github.com/SAP/jenkins-library/pkg/mock"
+	"github.com/stretchr/testify/assert"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+)
+
+type mockLintUtilsBundle struct {
+	*mock.FilesMock
+	execRunner *mock.ExecMockRunner
+}
+
+func (u *mockLintUtilsBundle) getExecRunner() execRunner {
+	return u.execRunner
+}
+
+func (u *mockLintUtilsBundle) getGeneralPurposeConfig(configURL string) error {
+	u.AddFile(filepath.Join(".pipeline", ".eslintrc.json"), []byte(`abc`))
+	return nil
+}
+
+func newNpmMockUtilsBundle() mockLintUtilsBundle {
+	utils := mockLintUtilsBundle{FilesMock: &mock.FilesMock{}, execRunner: &mock.ExecMockRunner{}}
+	return utils
+}
+
+type npmExecuteOptions struct {
```

Comment:
Isn't that the same struct which is used everywhere elese. Why do we have to define it again?